### ISSUE TITLE
About testing.

### ISSUE THREAD

**Issue by haoweiz23** [JuanLuisGonzalez/FAL_net#3]
Hi, appreciate your fantastic work. I got some problems in testing with 'Test_KITTI.py'. I found I did not get the directory 'proj_depth' and 'velodyne_raw' in 'Kitti_eigen_test_improved.py' and 'npy' files in 'Kitti_eigen_test_original.py' neither.

I guess  there are some differnce between your data files orignization and monodepth2. I can directly test with 'eval_depth.py' in monodepth2. So could you please provide more details about your data orgnization and how to reproduce your results in Table2 ? Thanks.

**Comment by guogangok** [comment]
Yes, it seems the .npy files must exist in the same dir of the corresponding images.

**Comment by xingyuuchen** [comment]
Neither do I...

**Comment by xingyuuchen** [comment]
> 

Hi!
Did you mean that you already reproduced the result by using 'eval_depth.py' in Monodepth2?

**Comment by haoweiz23** [comment]
I can reproduce Monodepth2 via their official code. There are many works improved based on their code.

**Comment by xingyuuchen** [comment]
>  I can reproduce Monodepth2 via their official code. There are many works improved based on their code.

Oh, I misunderstood. I took it for granted that you reproduced the result of Fal-net by using 'eval_depth.py' from Monodepth2.🤣

**Comment by xingyuuchen** [comment]
So you neither reproduced the result of Fal_net until now, did you?😑

**Comment by haoweiz23** [comment]
> So you neither reproduced the result of Fal_net until now, did you?😑

No updates after this issue opened.
